packages/debugger/public/frames.js/index.d.ts

```diff
@@ -0,0 +1,124 @@
+type JsonObject = { [Key in string]: JsonValue } & {
```

Comment:
i am concerned about this going out of date when a change is made in the other types files. is there a way we can generate this file on build perhaps?

Reply:
I was trying to find a way, but the issue is that it always produces definition file per js/ts file, we need to merge them to only one file.

Reply:
Ok I found a way how to do this.